package.json

```diff
@@ -599,34 +599,35 @@
 		"test": "npm run compile && node ./out/test/runTest.js"
 	},
 	"devDependencies": {
```

Comment:
I updated some of the packages.

Reply:
Super excited about the one to TypeScript, thank you Albert!

Your doing some pretty heavy foundational stuff. So, I am going to locally backup the stuff I've been working on, wait for the dust to settle on this PR and start from a fresh main branch before I author another.

Reply:
Not necessary, but the "@types/chai" would be a nice to have. Now that everything else has rolled onto newer versions I don't suspect I'll have the same issues temporarily putting it in there locally though.

Reply:
@JD-Howard Don't hesitate to add this. I don't want to muddy the PR with this unrelated change.